templates/partials/devel_build.dockerfile.j2

```diff
@@ -10,6 +10,8 @@ ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/opt/conda/envs/rapids/lib
 {% for lib in RAPIDS_LIBS %}
 RUN cd ${RAPIDS_DIR}/{{ lib.name }} && \
   source activate rapids && \
+  env && \
```

Comment:
Do we need to keep `env` for each build? It does add an extra amount of logging, would moving this up out of the loop be sufficient?

Reply:
Removed `env` and `ccache` as they are unnecessary. The `ccache` command might also cause some issues in the `quick` images if they aren't updated to FROM the new devel images in time.